test/case/infix_interfaces/bridge_fwd_dual_dut.py

```diff
@@ -0,0 +1,153 @@
+                    },
+                    {
+                        "name": tport12,
+                        "type": "iana-if-type:ethernetCsmacd",
+                        "enabled": True,
+                        "infix-interfaces:bridge-port": {
+                            "bridge": "br0",
+                        }
+                    }
+                ]
+            }
+        })
+
+        dut2.put_config_dict("ietf-interfaces", {
+            "interfaces": {
+                "interface": [
+                    {
+                        "name": "br0",
+                        "type": "iana-if-type:bridge",
+                        "enabled": True,
+                        "bridge": {
+                            "vlans": {
+                                "vlan": [
+                                    {
+                                        "vid": 10,
+                                        "untagged": [ tport20, tport21 ],
+                                        "tagged":   [ "br0", tport22 ]
+                                    }
+                                ]
+                            }
+                        }
+                    },
+                    {
+                        "name": tport20,
+                        "type": "iana-if-type:ethernetCsmacd",
+                        "enabled": True,
+                        "infix-interfaces:bridge-port": {
+                            "pvid": 10,
+                            "bridge": "br0"
+                        }
+                    },
+                    {
+                        "name": tport21,
+                        "type": "iana-if-type:ethernetCsmacd",
+                        "enabled": True,
+                        "infix-interfaces:bridge-port": {
+                            "pvid": 10,
+                            "bridge": "br0"
+                        }
+                    },
+                    {
+                        "name": tport22,
+                        "type": "iana-if-type:ethernetCsmacd",
+                        "enabled": True,
+                        "infix-interfaces:bridge-port": {
+                            "bridge": "br0",
+                        }
+                    }
+                ]
+            }
+        })
+
+    with test.step("Ping host:data20 with IP 10.0.0.3 and host:data21 with IP 10.0.0.4 from host:data11 with IP 10.0.0.2"):
+        
+        _, hport11 = env.ltop.xlate("host", "data11")
+        _, hport20 = env.ltop.xlate("host", "data20")
+        _, hport21 = env.ltop.xlate("host", "data21")
+
+        with infamy.IsolatedMacVlan(hport11) as ns11, \
+             infamy.IsolatedMacVlan(hport20) as ns20, \
+             infamy.IsolatedMacVlan(hport21) as ns21:
+
+            verify(ns11.addip("10.0.0.2"))
+            verify(ns20.addip("10.0.0.3"))
+            verify(ns21.addip("10.0.0.4"))
+
+            verify(ns11.ping("10.0.0.3"))
+            verify(ns11.ping("10.0.0.4"))
```

Comment:
With the `check=True` thing we added to `ping`, you can drop the `verify`, right?

Reply:
yeppp. "verify" removed